packages/core/addon/components/cell-renderers/dimension.js

```diff
@@ -58,12 +72,17 @@ export default Component.extend({
   /**
    * @property {String} value - value that should be displayed in table cell
    */
-  value: computed('descField', 'idField', 'data', function() {
-    let descField = get(this, 'descField'),
+  value: computed('dimensionField', 'descField', 'idField', 'data', function() {
+    let dimensionField = get(this, 'dimensionField'),
+      descField = get(this, 'descField'),
       idField = get(this, 'idField'),
       data = get(this, 'data');
```

Comment:
Do we need to fetch these values if we don't use them?

Reply:
you mean fetch the rest only if `dimensionField` is null?